evap/evaluation/tools.py

```diff
@@ -116,3 +116,47 @@ def get_parameter_from_url_or_session(request, parameter, default=False):
 def translate(**kwargs):
     # get_language may return None if there is no session (e.g. during management commands)
     return property(lambda self: getattr(self, kwargs[get_language() or 'en']))
+
+
+def round_datetime(dt=None, round_to=60):
```

Comment:
I think, since we're not usually doing this in a closed loop a lot of times, I'd prefer having `round_to` taking a string that would either be "seconds", "minutes" or "hours" and you would then do a dictionary lookup to find the corresponding number you need. This would make calls much more readable (imo)